.github/workflows/build-envoy-image-ci.yaml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Prep for build
         run: |
           echo "${{ github.event.pull_request.head.sha }}" >SOURCE_VERSION
-          echo "ENVOY_VERSION=$(cat ENVOY_VERSION)" >> $GITHUB_ENV
+          echo "ENVOY_MINOR_RELEASE=$(cat ENVOY_VERSION | sed 's/envoy-\([0-9]\+\.[0-9]\+\)\..*/v\1/')" >> $GITHUB_ENV
```

Comment:
Looks like we did not use `ENVOY_VERSION` for anything... so OK for removing it.

Reply:
good point :100: